visidata/vd.py

```diff
@@ -1910,16 +2110,19 @@ def run(sheetlist=[]):
         print(ret)
 
 def curses_main(_scr, sheetlist=[]):
+    """Populate VisiData object with sheets from a given list."""
     for vs in sheetlist:
         vd().push(vs)  # first push does a reload
     return vd().run(_scr)
 
-g_globals = None
+g_globals = None # TODO: If we expect a dict, why not `dict()` here?
```

Comment:
No change. I just mean that assigning `g_globals = dict()` will still be `False` (the same as `g_globals = None`), but the reader will know that a dict is intended.